amostra/objects.py

```diff
@@ -39,8 +39,9 @@ def __init__(self, _amostra_client, *args, **kwargs):
     def __new__(cls, *args, **kwargs):
         # Configure _validate_with_jsonschema to validate all traits.
         trait_names = list(cls.class_traits())
-        cls._validate = validate(*trait_names)(_validate_with_jsonschema)
-        return super().__new__(cls, *args, **kwargs)
+        instance = super().__new__(cls, *args, **kwargs)
+        instance._validate = validate(*trait_names)(_validate_with_jsonschema)
```

Comment:
I think think that this change makes the validation not happen at al!  The `validate` method produces a descriptor which needs to be in place for the `super().__new__(...)` to find them and do something about it.

Are there any tests we we are sure that we do reject invalid documents?

Reply:
Yes, I think @tacaswell is right. It would be good to add a test that uses `pytest.raises` to ensure that validation is running and correctly raising an error on invalid inputs.

Reply:
I agree with you that validation wasn't triggered.